### PR TITLE
Merge env-standalone to babel-standalone

### PR DESCRIPTION
**Pull request by JLHwung**
| Q                        | A 
| ------------------------ | ---
| Minor: New Feature?      | Yes
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  | Add more devDependencies to babel-standalone
| License                  | MIT

This PR registers `env` in `babel-standalone` and copies the plugins in `babel-preset-env-standalone` to `babel-standalone`. By doing so babel-standalone will now includes all the functions in env-standalone, and we intend to remove env-standalone on 8.x.

The following size change is produced from [CI Artifacts before](https://app.circleci.com/jobs/github/babel/babel/13289) and [CI Artifcats after](https://app.circleci.com/jobs/github/babel/babel/13293)

| | **Previous Size** | **New Size** | **Change** |
| --- | --- | --- | --- |
| babel.js | 2.4 MB | 2.9 MB | +20% |
| babel.min.js | 1.2 MB | 1.5 MB | +25% |
| babel-preset-env.js | 3.0 MB | 3.0 MB | - |
| babel-preset-env.min.js | 1.3 MB | 1.3 MB | - |

**Edit by @nicolo-ribaudo**
The previous table should be read as this. These sizes refer to the whole bundle including all the plugins + preset-env.

|  | **Previous Total Size** | **New Total Size** | **Change** |
| --- | --- | --- | --- |
| Not minified | 5.4 MB | 2.9 MB | -46% |
| Minified | 2.5 MB | 1.5 MB | -40% |

### CODE REVIEW

**Comment by nicolo-ribaudo** [comment]
Could you post the bundle size differences?

**Comment by JLHwung** [comment]
@nicolo-ribaudo Yep, I _am_ preparing.

File: packages/babel-standalone/test/babel.js [hunk 1]
```diff
@@ -110,6 +110,61 @@
       ).toThrow(/Invalid plugin specified in Babel options: "lolfail"/);
     });
 
+    describe("env preset", () => {
```
Comment (JLHwung) [inline packages/babel-standalone/test/babel.js]: The [REPL Build](https://babeljs.io/repl/build/13293) is not working. I will investigate it today.

**Comment by JLHwung** [comment]
@nicolo-ribaudo Note that both `nullish-coalescing` and `optional-chaining` are merged to `preset-env`, this PR is still good to go because those two features are added to both sides. This PR cares only those exclusively available in `-standalone`.

**Pull request merged**